src/main/java/seedu/address/logic/commands/schedulercommands/DeleteEventCommand.java

```diff
@@ -23,6 +23,7 @@
     public static final String MESSAGE_DELETE_EVENT_SUCCESS = "Deleted event: %1$s";
     private final Index index;
     public DeleteEventCommand(Index index) {
+        assert (index != null);
```

Comment:
Can use requireNonNull since it is already provided by the util package.

Reply:
I think its ok to use both, and assert might be better since the tutorial asked us to use asserts.